README.md

```diff
@@ -181,6 +184,29 @@ Note that our CI system checks that code formatting is consistent, and the build
 
 Also note that we run some `clang-tidy` linting over the code as well (see `.clang-tidy` for the list of rules enforced).  You can run `./scripts/tidy.sh` over the code before committing to ensure you haven't added any of the common problems we check for (Note: `./scripts/tidy.sh` requires the exitence of a `compile_commands.json` database.  You can generate this file by running `bear make` instead of just `make`.  The `bear` tool is installable on Ubuntu-based systems with `apt-get install bear`, and on macOS with `brew install bear`).
 
+Using the Node.js Bindings
+--------------------------
+
+The Node.js bindings are still under construction. We are working on building binaries for as many environments as possible, but they may not all be available yet. The first functionality that we are exposing is `route`, but we plan on exposing more of the tyr actor functions. Right now, the input and the output are both strings - THAT WILL CHANGE. We plan on ingesting and producing protobufs.
+
+The Node.js bindings provide read-only access to the routing engine. You can install the Node.js bindings via npm install valhalla or from this repository either via
+
+`$npm install`
+which will check and use pre-built binaries if they're available for this release and your Node version, or via
+
+`$npm install --build-from-source`
+to always force building the Node.js bindings from source.
+
+Example of using in a node project:
+```js
+var valhalla = require('valhalla');
+var router = new valhalla.Actor(configString);
```

Comment:
Soooo. I have been wondering whether we should rename `Actor` to `Router` instead?
That would change the Node code to 
```
var router = new valhalla.Router(configString);
```
which makes it look more intuitive. I think the term `Actor` comes the fact that valhalla has a `Worker/Actor` design. But that design is pretty opaque to the Node binding user and I wonder whether the user should even know and care?

Reply:
Agreed that the internals of worker/actor are maybe not important for node users. Could this be simplified to `var valhalla = new Valhalla(cfg)`?

Reply:
regarding the name, it doesnt just `route`, the `actor` does all the `actions`: height/locate/route/optimize/isochrone...

there is a technical reason why there should be a module level configuration although we could ignore this if we are ok with telling a small lie about what is going on behind the scenes.

there is one thing (logging) that is configured in a singleton (so globably once). the rest of the stuff in the config gets parsed every time you instantiate an actor. to properly adhere to this, what we could do is:

    var valhalla = require('valhalla')(configString);
    var actor = new valhalla.Actor(); //uses globally configured configString internally

this stops you from being able to reconfigure an actor on the fly. so we could do as @danpaz was saying and allow it be reconfigured on the fly with the knowledge that (small lie) logging will not get reconfigured.

Reply:
I do like the idea of hiding the term `Actor` from the user, since it will have little context to them. I wonder if we could do:

```
var Valhalla = require('valhalla')(configString);
var valhalla = new Valhalla();
valhalla.route(blah);
```

Reply:
I'll play around and see if I can get something like this working ☝️